.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,17 @@
+name: Nightly
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+jobs:
+  test-gke:
+    name: Request GKE test cluster
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Just wondering if we need this? We don't seem to be doing anything with this step.

Reply:
It was used, but not anymore, will put it back in if needed.